wifiphisher/macmatcher.py

```diff
@@ -0,0 +1,45 @@
+import os
+from constants import *
+
+class MACMatcher():
+    """
+        This class is using Organizationally Unique Identifiers (OUIs) 
+        in order to match MAC addresses of devices to its vendord  
+
+        See http://standards.ieee.org/faqs/OUI.html
+    """
+    def __init__(self, mac_prefix_file=MAC_PREFIX_FILE):
```

Comment:
Personally I don't think we need to pass the mac file as an argument since it will always be the same. @sophron let me know if you think differently.